ush/templates/FV3LAM_wflow.xml

```diff
@@ -117,7 +120,7 @@ tasks; and the "FCST" type is used for the RUN_FCST_TN task.
 <!ENTITY RSRV_FCST    "<account>&ACCOUNT;</account><queue>&QUEUE_FCST;</queue>">
 {%- endif %}
 {%- if partition_analysis is not none %}
-<!ENTITY RSRV_ANALYSIS    "<account>&ACCOUNT;</account><queue>&QUEUE_ANALYSIS;</queue><partition>{{ partition_analysis }}</partition>">
+<!ENTITY RSRV_ANALYSIS    "<account>&ACCOUNT;</account><queue>&QUEUE_ANALYSIS;</queue><partition>{{ partition_analysis }}</partition><native>&RRFS_RESERVE;</native>">
```

Comment:
Just curious whether missing "RRFS_RESERVE" causes issues? It looks like ”rth" and "rt" queues only work for reservation nodes. If there is a problem, we need to add it to "RSRV_FCST" as well.